languages/csharp/reference/exercise-concepts/two-fer.md

```diff
@@ -0,0 +1,15 @@
+# Concepts of Two Fer
+
+[Example implementation](https://github.com/exercism/csharp/blob/master/exercises/two-fer/Example.cs)
+
+## General
```

Comment:
This is great! The only things that I would add are the "strings" and "function arguments" (of which optional arguments is a specialized Concept) Concepts. Does that make sense?

Reply:
Sure, thank you for your review. I will add these.